2015-04-30  Yi Wang  <yiwa@lnxsrv05.seas.ucla.edu>

	* timespec.h: change the timespec.h function because it may cause overflow.
--- timespec.h	2015-04-30 22:25:13.835515000 -0700
+++ ./coreutils-with-bug/lib/timespec.h	2005-09-21 23:05:39.000000000 -0700
@@ -44,17 +44,9 @@
    Assume the nanosecond components are in range, or close to it.  */
 static inline int
 timespec_cmp (struct timespec a, struct timespec b)
-{ 
-  if (b.tv_sec>a.tv_sec)
-    return -1;
-  else if (a.tv_sec>b.tv_sec)
-    return 1;
-  else
-  {
+{
   int diff = a.tv_sec - b.tv_sec;
   return diff ? diff : a.tv_nsec - b.tv_nsec;
-  }
-  
 }
 
 # if ! HAVE_DECL_NANOSLEEP
